packages/reports/addon/models/bard-request/request.js

```diff
@@ -2,21 +2,21 @@
  * Copyright 2018, Yahoo Holdings Inc.
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  */
-
-import Ember from 'ember';
 import DS from 'ember-data';
 import MF from 'model-fragments';
 import isEqual from 'lodash/isEqual';
 import { validator, buildValidations } from 'ember-cp-validations';
 import Interval from 'navi-core/utils/classes/interval';
+import { A, makeArray } from '@ember/array';
```

Comment:
Small thing: we usually rename `A` to `arr`

```js
import { A as arr } from '@ember/array';
```
https://github.com/yahoo/navi/search?q=A%3Aarr&unscoped_q=A%3Aarr

Reply:
👍